.github/workflows/flake.yml

```diff
@@ -0,0 +1,39 @@
+name: Flake
+
+# Run flake on all PRs.  This ignores many pre-existing errors however should catch
+# regressions and allows us to remove ignores and results issues iteratively.
+
+on:
+  pull_request:
+    paths:
+      - '**/*.py'
+
+jobs:
+  flake8-lint:
+    runs-on: ubuntu-latest
+    name: Lint
+    steps:
+      - name: Check out source repository
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Set up Python environment
+        uses: actions/setup-python@v1
+        with:
+          python-version: "3.8"
+      - name: Add parser
+        run: echo -n "::add-matcher::ci/daos-flake-matcher.json"
+      - name: Add whitespace parser
+        run: echo -n "::add-matcher::ci/daos-flakew-matcher.json"
+      - name: Add error parser
+        run: echo -n "::add-matcher::ci/daos-flakee-matcher.json"
+      - name: flake8 Lint
+        uses: py-actions/flake8@v2
+        with:
+          # W503 and W504 are related as they conflict.  W504 is the preferred style, however we
+          # have more code that matches W503 so accept either at this point.
+          ignore: "F841,W503,W504,E12,E2,E302,E402,E714,E731"
+          exclude: "src/control/vendor,src/client/pydaos/raw"
+          # Max line length is 100, and pylint will warn about lines longer than this, but some
+          # existing lines are longer so allow them here.
+          max-line-length: "133"
```

Comment:
If our limit really is 100, shouldn't these too-long ones be a "to-fix" kind of item?

Reply:
This is the storage_estimator code, which has numerous problems - it was somehow ignored by pylint when added so there are lots of major issues with it.

Reply:
The concern of course is this letting in more >100 character lines.

Reply:
As per the comment pylint will still warn on that so there's no regression here, but this could be stricter yes.